### PR TITLE
Jest - header, hover tests, Attraction tests

### PR DESCRIPTION
**Pull request by this-Cunningham**
lots of tests written

### CODE REVIEW

**Comment by this-Cunningham** [comment]
[Jest Tests](https://trello.com/c/GilWCfFC/56-jest-tests)

**Comment by this-Cunningham** [comment]
[Jest Tests](https://trello.com/c/GilWCfFC/56-jest-tests)

File: src/styles.css [hunk 1]
```diff
@@ -55,6 +55,7 @@ h3 {
   margin: 0;
 }
 
+
```
Comment (pachown) [inline src/styles.css:58]: misclick mistake

**Review by pachown — APPROVED** [review]
The tests aren't passing in CircleCI. Its hard to tell on my end if they aren't passing at all or if Circle CI isn't set up correctly. From my limited knowledge of these enzyme tests it looks good though. Really great progress

**Pull request closed without merging**